lib/open_api_spex/discriminator.ex

```diff
@@ -20,14 +20,14 @@ defmodule OpenApiSpex.Discriminator do
   specification of an alternative schema based on the value associated with it.
   """
   @type t :: %__MODULE__{
-    propertyName: String.t,
-    mapping: %{String.t => String.t} | nil
-  }
+          propertyName: String.t(),
```

Comment:
I think it's a good idea to restrict reformatting changes to only the code we change. It's an open source project for one, and also when we merge this back in, it's going to be huge. It would be good to minimize the size and potential conflicts.